pkg/controller/migmigration/task.go

```diff
@@ -535,7 +535,18 @@ func (t *Task) Run(ctx context.Context) error {
 			return liberr.Wrap(err)
 		}
 		if backup == nil {
-			return errors.New("Backup not found")
+			t.Owner.Status.StageCondition(migapi.Running)
```

Comment:
probably want to create a function for this code since it is almost similar at 4 places